packages/components/src/number-control/stories/index.story.tsx

```diff
@@ -14,7 +14,8 @@ import { useState } from '@wordpress/element';
 import NumberControl from '..';
 
 const meta: Meta< typeof NumberControl > = {
-	title: 'Components (Experimental)/NumberControl',
+	title: 'Components/Selection & Input/Common/NumberControl',
```

Comment:
Changed from `experimental` to `stable` as per the sitemap. If not this can be reverted to use `experimental` only.